src/frequenz/sdk/actor/_config_managing.py

```diff
@@ -64,7 +64,7 @@ def _read_config(self) -> dict[str, Any]:
             with open(self._conf_file, "rb") as toml_file:
                 return tomllib.load(toml_file)
         except ValueError as err:
-            logging.error("Can't read config file, err: %s", err)
+            logging.error("%s: Can't read config file, err: %s", self, err)
```

Comment:
would it make sense to log the exception instead? I mean `logging.exception(...)`

Reply:
I don't think so, we just want to know that a toml field value is invalid, having the whole stack trace doesn't help much and adds quite a bit of noise, right?

Reply:
Ok. Actually the exception will be logged as this method raises an error that only the actor will capture and log the exception before getting restarted. So we don't need to do it here